compiler/ballerina-lang/src/test/java/io/ballerina/projects/FileSystemRepositoryTests.java

```diff
@@ -39,7 +39,7 @@
  */
 public class FileSystemRepositoryTests {
 
-    class MockFileSystemRepository extends FileSystemRepository {
+    static class MockFileSystemRepository extends FileSystemRepository {
```

Comment:
Shall we make this `private`? I think this is not used outside of the `FileSystemRepositoryTests`. 
We can check the same in other places as well.